testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/theme/ThemeResourceProviderTest.java

```diff
@@ -50,4 +51,18 @@ public void getResourceAsStream() {
         });
     }
 
+    @Test
+    public void getMessages() {
+        testingClient.server().run(session -> {
+            try {
+                ThemeProvider extending = session.getProvider(ThemeProvider.class, "extending");
+                Theme theme = extending.getTheme("base", Theme.Type.LOGIN);
+                System.out.println(theme.getMessages("messages", Locale.ENGLISH));
```

Comment:
Please remove this line as we don't want this output when running tests

Reply:
done